config/routes.rb

```diff
@@ -0,0 +1,14 @@
+Rails.application.routes.draw do
+  get 'post/new'
+  get 'post/create'
+  get 'post/index'
+  get 'static_pages/home'
+  root "post#index"
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :sessions, only: [:new, :create, :destroy]
```

Comment:
Another way to do this job with custom naming

```ruby
get '/login', to: 'sessions#new'
post '/login', to: 'sessions#create'
delete '/logout', to: 'sessions#destroy'
```
In that case there no need any resources for the session and also no need any match. As simple as it is.